src/PagerAndroid.js

```diff
@@ -33,8 +33,7 @@ export default class PagerAndroid<T: *> extends React.Component<Props<T>> {
 
   componentDidUpdate(prevProps: Props<T>) {
     if (
-      prevProps.navigationState.routes.length !==
-        this.props.navigationState.routes.length ||
+      prevProps.navigationState.routes !== this.props.navigationState.routes ||
```

Comment:
Hey, why did we need this change?

Reply:
Does the length change over time? Aren't the references different when the routes change?

Reply:
@horodnicdragos the length could change if the user is adding/removing tags dynamically. the routes array could change even if the number of routes didn't change, but say the user added a new property/changed some value in a specific route. we don't need to update the scroll position in that case.

Reply:
I understand now, it makes sense. Let me revert those changes.

Reply:
No need. Already did it and merged.